x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -49,6 +49,25 @@ static DeprecationIssue oldIndicesCheck(IndexMetadata indexMetadata, ClusterStat
         return null;
     }
 
+    static DeprecationIssue ignoredOldIndicesCheck(IndexMetadata indexMetadata, ClusterState clusterState) {
+        // TODO: this check needs to be revised. It's trivially true right now.
```

Comment:
Is this comment still true?

Reply:
That comment was originally put in by core when this check first got added... I think you are right, it's evolved a lot now so I'm going to take it out.